editors/emacs/swarm-mode.el

```diff
@@ -25,18 +25,74 @@
 
 (setq swarm-font-lock-keywords
       (let* (
-             ;; We should figure out how to autogenerate these, so we don't have
-             ;; to edit the emacs mode every time we add new commands.
+             ;; Generate the current keywords with:
+             ;; cabal run swarm:swarm -- generate editors --emacs
              (x-keywords '("def" "end"))
-             (x-builtins '("if" "run" "return" "try" "raise" "force" "fst" "snd"))
-             (x-commands
-              '("noop" "wait" "selfdestruct" "move" "turn" "grab" "place" "give"
-                "install" "make" "build" "salvage" "reprogram" "knows"
-                "say" "log" "view" "appear" "create" "getx" "gety"
-                "blocked" "scan" "upload" "ishere" "whoami"
-                "random" "not"
-                "left" "right" "back" "forward" "north" "south" "east" "west" "down"
-                ))
+             (x-builtins '(
+               "if"
```

Comment:
@byorgey or @TristanCacqueray, do either of you use EMacs and could check this works, please?

Reply:
Works great!